dashboard/views.py

```diff
@@ -313,7 +313,8 @@ def grade_distribution(request, course_id=0):
         from user where course_id=%(course_id)s and enrollment_type='StudentEnrollment';
                     """
     df = pd.read_sql(grade_score_sql, conn, params={"current_user": current_user, 'course_id': course_id})
-    if df.empty or df['current_grade'].isnull().all():
+    if df.empty or df['current_grade'].isnull().all() or df.count().current_grade < 6:
+        logger.info(f"Not enough students grades {df.count().current_grade} in a course {course_id} to show the view")
```

Comment:
Maybe modify the comment so the number of grades are in parentheses? Something like this:
`"Not enough students (only {df['current_grade'].count()}) in a course ..."`